wiki/Main_page/en.md

```diff
@@ -17,29 +17,29 @@ Important articles and guides that will help you dive into osu!. In case anythin
 
 [Rules](/wiki/Rules) • [Installation](/wiki/Client/Installation) • [Registration](/wiki/Registration) • [Help Centre](/wiki/Help_centre)
 
-[Chat console](/wiki/Client/Interface/Chat_console) • [BanchoBot](/wiki/BanchoBot) • [FAQ](/wiki/FAQ) • [Guides](/wiki/Guides) • [Glossary](/wiki/Sitemap)
```

Comment:
The sitemap was also under "Wiki and meta pages" and I thought I would leave it there and take it out of "Getting Started" since this huge list doesn't seem to serve the purpose to help out new players.
I moved Bancho Bot to the "Game Client" section since there should still be a way to get to the Bancho Bot page.